src/components/IntroductionPage/index.js

```diff
@@ -21,12 +22,15 @@ const IntroductionPage = () => {
         dangerouslySetInnerHTML={{__html: introductionPageContent}}
       />
 
-      <Link to="/startpagina" component={ButtonLink} appearance="primary-action-button">
-        <FormattedMessage
-          description="Button text for link to continue from introduction page to start page"
-          defaultMessage="Continue ➜"
-        />
-      </Link>
+      <span className="openforms-start-link">
+        <Link to="/startpagina" component={ButtonLink} appearance="primary-action-button">
+          <FormattedMessage
+            description="Button text for link to continue from introduction page to start page"
+            defaultMessage="Continue"
+          />
+          <FAIcon icon="arrow-right-long" extraClassName="openforms-start-link--icon" />
```

Comment:
```suggestion
          <FAIcon icon="arrow-right-long" extraClassName="openforms-start-link__icon" />
```
`__` signifies a child/element inside the block/component, `--` is for modifiers

Reply:
ah shit, my bad 🙈 . I'll change it

Reply:
With the flex-box change, this class will be removed